lib/devops.ts

```diff
@@ -43,6 +43,33 @@ export abstract class DevOpsBaseClient implements IDevOpsClient {
 
   abstract async fetchPullRequests(): Promise<any>;
 
+  filterPRs(devOpsResponse: DevOpsResponse): DevOpsResponse {
+    if (!this.devOpsTeam.filters) {
```

Comment:
Probably could also be 
```suggestion
    if (!this.devOpsTeam.filters && this.devOpsTeam.filters.length > 0) {
```
to check for zero length filters

Reply:
do you mean `&& this.devOpsTeam.filters.length === 0)`?

Reply:
A yeah, double negative.

Reply:
Best of all worlds https://github.com/paxos/motivatron/pull/3/commits/fb2c1272c07149bb0d4fc972c1f88a6b9600f977:
```
 if (!Array.isArray(this.devOpsTeam.filters)) {
```